_includes/manuals/1.11/3.6_upgrade.md

```diff
@@ -7,6 +7,19 @@ When upgrading across more than one version it is recommended to upgrade to each
 intermediate version and follow all upgrade instructions for the previous versions
 before updating to {{page.version}}.
 
+#### Known issues
+Currently, [tfm-rubygem-ovirt_provision_plugin prevents](https://github.com/theforeman/ovirt_provision_plugin/issues/5) 
+the upgrade process to finish correctly and needs to be removed if it was installed 
+previously in Foreman versions <= 1.10.X:
```

Comment:
add "on EL6 or EL7.", as the instructions here are specifically for it and no package is provided for Debian/Ubuntu